### PR TITLE
Banner: remove Unhandled Rejections Survey

### PR DESCRIPTION
**Pull request by mmarchini**
The survey ends on Aug 24th, remove the banner so folks don't try to
respond to it after it closes.

### CODE REVIEW

**Comment by mmarchini** [comment]
We should wait until Sunday to land this

**Comment by mmarchini** [comment]
cc @nodejs/website I'd like to merge this today

**Review by nschonni — APPROVED** [review]

**Pull request merged**
